frontend/src/containers/AdmissionsContainer/InnerTable.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo } from "react";
+import { useTable, useSortBy, useBlockLayout } from "react-table";
+import { innerCols } from "./Columns";
+
+const AdmissionsInnerTable = ({ groupApplications }) => {
+  const columns = useMemo(() => innerCols, []);
```

Comment:
Is `useMemo` needed here? Considering the value never changes I don't think it's necessary

Reply:
According to the useTable docs columns must be memoized (https://react-table-v7.tanstack.com/docs/api/useTable).